app/components/referee_interface/reference_review_component.rb

```diff
@@ -1,13 +1,16 @@
 module RefereeInterface
   class ReferenceReviewComponent < ViewComponent::Base
-    def initialize(reference:, token_param: nil, editable: true)
+    def initialize(reference:, application_form:, token_param: nil, editable: true)
       @reference = reference
       @editable = editable
       @token_param = token_param
+      @application_form = application_form
```

Comment:
I don't think we need to pass the application form into this component. We can get what we need from the reference (`reference.application_form.full_name`).

Reply:
We could get the `application_form` from the reference, but I would prefer less calls to the database.

Since we already have the `application_form` available in the component, passing it in directly is more efficient.

The `application_form` may be useful to have for future work too.

Reply:
Okeedokee